subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolveengine/artifact/DefaultArtifactSet.java

```diff
@@ -94,6 +94,6 @@ public Set<ResolvedVariant> getVariants() {
 
     @Override
     public Set<ResolvedVariant> getAllVariants() {
-        return componentArtifactResolveVariantState.getAllVariants();
+        return allVariants.getAllVariants();
```

Comment:
🧐 I'm not sure this rename makes anything clearer.  The other fields are all just named for their types, this one stands out, but not in a good way.  I think a javadoc comment on the param or field might be a better way to communicate the usage.